sebs/experiments/perf_cost.py

```diff
@@ -46,21 +46,22 @@ def prepare(self, sebs_client: "SeBS", deployment_client: FaaSSystem):
         self._benchmark = sebs_client.get_benchmark(
             settings["benchmark"], deployment_client, self.config
         )
-        self._function = deployment_client.get_function(self._benchmark)
+        self._functions = deployment_client.get_function(self._benchmark, 3)
         # prepare benchmark input
         self._storage = deployment_client.get_storage(replace_existing=self.config.update_storage)
         self._benchmark_input = self._benchmark.prepare_input(
             storage=self._storage, size=settings["input-size"]
         )
+        for i in range(len(self._functions)):
```

Comment:
I imagine this seems more complex than anticipated since cloud platforms expose an HTTP trigger, and the local deployment would have a different HTTP address for each instance. We don't want to have to change every experiment - we want to hide this complexity behind the trigger.

The simplest solution would be to implement a new `sebs.Local.ScalableHTTPTrigger` (or something similar) that would allocate/deallocate container instances, and redirect invocations to the proper HTTPTrigger.